bar.rb

```diff
@@ -0,0 +1,3 @@
+def camelCase()
```

Comment:
Omit the parentheses in defs when the method doesn't accept any arguments.<br>Use snake_case for method names.